lib/rules/web/admin_console/4.9/projects.xyaml

```diff
@@ -77,13 +77,34 @@ check_cpu_data_for_one_project_in_table:
     resource_name: <project_name>
     data: cores
   action: check_resource_data_in_table
+set_strings_in_project_dropdown_filter_box:
+  elements:
+  - selector: &dropdown_filter
+      xpath: //input[@data-test='dropdown-text-filter']
+    op: clear
+  - selector:
+      <<: *dropdown_filter
+    op: send_keys <project_name>
```

Comment:
is this action the same thing with `set_strings_in_dropdown_filter_box` ?
```
set_strings_in_dropdown_filter_box:
  elements:
  - selector: &dropdown_filter
      xpath: //input[@data-test-id='dropdown-text-filter']
    op: clear
  - selector:
      <<: *dropdown_filter
    op: send_keys <filter_text>
```

Reply:
Different on XPath part, update input from @data-test-id to @data-test, then it will not impact other cases